Genfare/ViewControllers/Store/GFTicketCardSeletionViewController.swift

```diff
@@ -250,24 +253,36 @@ class GFTicketCardSeletionViewController: UIViewController,UITableViewDelegate,U
             
             present(alert, animated: true, completion: nil)
         }
+
         
             self.OrderProducts()
     
         
         
     }
     func pushToWebPage(){
+
         let navController = UIStoryboard(name: "Payment", bundle: nil).instantiateViewController(withIdentifier: "GFPurchaseWebViewController") as? GFPurchaseWebViewController
```

Comment:
Use guard let here so that you done need to force unwrap navController

Reply:
Done.